http-server/src/main/java/org/triplea/server/error/reporting/CreateIssueStrategy.java

```diff
@@ -34,17 +32,12 @@
 
   @Override
   public ErrorUploadResponse apply(final ErrorReportRequest errorReportRequest) {
-    if (allowErrorReport.test(errorReportRequest.getClientIp())) {
-      final ErrorUploadResponse errorUploadResponse = sendRequest(errorReportRequest);
+    final ErrorUploadResponse errorUploadResponse = sendRequest(errorReportRequest);
 
-      errorReportingDao.insertHistoryRecord(errorReportRequest.getClientIp());
-      errorReportingDao.purgeOld(Instant.now().minus(2, ChronoUnit.DAYS));
+    errorReportingDao.insertHistoryRecord(errorReportRequest.getClientIp());
+    errorReportingDao.purgeOld(Instant.now().minus(365, ChronoUnit.DAYS));
```

Comment:
Why this change from 2 days to a year?

Reply:
The year limit is a bit arbitrary, was debating between 3 months, 6 months, 1 year, and 3 years.

Before the table was used for rate limiting, now it's only good for stats/audit. Because we no longer read the table, but append to it only, the length of the table is less significant. In other words, a large table would cause the read query to be slow, now it does not matter if it's large. It's a bit arbitrary how long to keep stats for. A year seems long enough for interesting data and data older than that is perhaps not needed and less interesting.